provisioning/service/tests/Config/TpmAttestationTests.cs

```diff
@@ -215,5 +250,44 @@ public void TpmAttestation_ThrowsOnDeserializationWithEmptyEndorsementKey()
             // act - assert
             TestAssert.Throws<ProvisioningServiceClientException>(() => Newtonsoft.Json.JsonConvert.DeserializeObject<TpmAttestation>(json));
         }
+
+        [TestMethod]
+        [TestCategory("DevService")]
+        public void TpmAttestation_SucceedOnDeserializationWithoutStorageRootKey()
+        {
+            // arrange
+            string endorsementKey = KEY;
+            string json =
+                "{" +
+                "  \"endorsementKey\":\"" + endorsementKey + "\"," +
+                "}";
+
+            // act
+            TpmAttestation tpmAttestation = Newtonsoft.Json.JsonConvert.DeserializeObject<TpmAttestation>(json);
+
+            // assert
+            Assert.AreEqual(endorsementKey, tpmAttestation.EndorsementKey);
+        }
+
+        [TestMethod]
+        [TestCategory("DevService")]
+        public void TpmAttestation_SucceedOnDeserializationWithEmptyStorageRootKey()
+        {
+            // arrange
```

Comment:
Too much code duplication. Compact within a single test please.

Reply:
Testing for multiple scenarios with deserialization. Will look into using parameterized test in the future